app/main.py

```diff
@@ -1,3 +1,39 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from typing import List
+
+from app.client_resources.Point2D import Point2d
+from app.client_resources.customer import Customer, ServerWithFuelData
+from app.shop_resources.shop import Shop
+from app.client_resources.car import Car
+
+
+def create_shops(shops: list[dict]) -> List[Shop]:
+    return [Shop(shop_data["name"],
+                 Point2d(shop_data["location"]),
+                 shop_data["products"]) for shop_data in shops]
+
+
+def process_customers_list() -> List[Customer]:
+    try:
+        with open("app/config.json", "r") as f:
+            world_data = json.load(f)
+    except FileNotFoundError as fnfe:
+        print(f"Loading problem, {fnfe} doesn't exist.")
+        raise fnfe
```

Comment:
fnfe is bad name for exception, use full names for errors
no need to make print before raise exception